templates/.github/workflows/ci.yml

```diff
@@ -67,15 +69,8 @@ jobs:
         - uses: actions/checkout@v2
         - name: Set IMAGE_TAG as env
           run: |
-            APP_NAME=<% .Name %>
-            IMAGE_TAG=${APP_NAME}-$(git rev-parse --short=7 ${{ github.sha }})
+            IMAGE_TAG=${{ env.APP_NAME }}-$(git rev-parse --short=7 ${{ github.sha }})
             echo "IMAGE_TAG=${IMAGE_TAG}" >> $GITHUB_ENV
-        - name: Set APP_NAME and NAMESPACE as env
```

Comment:
This step was now unnecessary - each step of the GitHub action also logs all the environment variables anyway, so keeping the `echo` statements would be redundant.